.github/workflows/k8s-patch-os-jobs.yml

```diff
@@ -0,0 +1,173 @@
+# The EC2 AMI for K8s needs to be updated to prevent it from becoming outdated. This workflow will run in the beginning of every month to replace the existing
+# instances with new ones
+name: Update K8s EC2 Instance
+
+on:
+  workflow_call:
+    inputs:
+      repo:
+        required: true
+        type: string
+      ec2_name:
+        required: true
+        type: string
+      language:
+        required: true
+        type: string      
+      
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  REPO: ${{ inputs.repo }}
+  EC2_NAME: ${{ inputs.ec2_name }}
+  LANGUAGE:  ${{ inputs.language }}
+  E2E_TEST_AWS_REGION: us-east-1
+  E2E_TEST_ACCOUNT_ID: ${{ secrets.APPLICATION_SIGNALS_E2E_TEST_ACCOUNT_ID }}
+  E2E_TEST_ROLE_NAME: ${{ secrets.APPLICATION_SIGNALS_E2E_TEST_ROLE_NAME }}
+
+jobs:
+  create-k8s-on-ec2:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate Testing ID
+        run: |
+          echo TESTING_ID="${{ github.run_id }}-${{ github.run_number }}" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ env.E2E_TEST_ACCOUNT_ID }}:role/${{ env.E2E_TEST_ROLE_NAME }}
+          aws-region: us-east-1
+
+      - name: Create New K8s on EC2
+        working-directory: .github/workflows/util
+        run: |
+          ./setup-k8s.sh ${{ env.TESTING_ID }} ${{ env.EC2_NAME }}-${{ env.LANGUAGE }} ${{ env.E2E_TEST_AWS_REGION }}
+
+      - name: Check if Secret Key Already Exist and If Not Create New Ones
+        run: |
+          SECRET_NAMES=(
+            "e2e-test/${{ env.REPO }}/${{ env.LANGUAGE }}-k8s-master-node-endpoint-pending-value"
+            "e2e-test/${{ env.REPO }}/${{ env.LANGUAGE }}-k8s-ssh-key-pending-value"
+            "e2e-test/${{ env.REPO }}/${{ env.LANGUAGE }}-k8s-master-node-endpoint"
+            "e2e-test/${{ env.REPO }}/${{ env.LANGUAGE }}-k8s-ssh-key"
+            "e2e-test/${{ env.REPO }}/${{ env.LANGUAGE }}-k8s-master-node-endpoint-temporary-storage"
+            "e2e-test/${{ env.REPO }}/${{ env.LANGUAGE }}-k8s-ssh-key-temporary-storage"
+          )
+
+          for SECRET_NAME in "${SECRET_NAMES[@]}"; do
+            echo "Checking secret: $SECRET_NAME"
+
+
+            if aws secretsmanager describe-secret --secret-id $SECRET_NAME; then
+              echo "Success: The secret exists."
+            else
+              echo "Secret does not exist. Creating new secret"
```

Comment:
```suggestion
              echo "Secret \"<INSERT THE SECRET NAME>\" does not exist. Creating it now"
```

Reply:
Hmm, the secret name is already called in line 64 `echo "Checking secret: $SECRET_NAME"`, but I guess it's more clear if we add it in the result message too